ArduPlane/Attitude.cpp

```diff
@@ -502,6 +501,11 @@ void Plane::stabilize()
     } else if (control_mode->is_vtol_mode() && !quadplane.tailsitter.in_vtol_transition(now)) {
         // run controlers specific to this mode
         plane.control_mode->run();
+        if (control_mode == &mode_qrtl &&
+            quadplane.poscontrol.get_state() <= QuadPlane::QPOS_AIRBRAKE) {
```

Comment:
```suggestion
            (quadplane.poscontrol.get_state() == QuadPlane::QPOS_APPROACH ||
            quadplane.poscontrol.get_state() == QuadPlane::QPOS_AIRBRAKE)) {
```
We have been caught out by less than in the past and found there are edge cases where we come through this in `QPOS_NONE`

Reply:
You could also put this in the run method for QRTL, its called on the line above.

https://github.com/ArduPilot/ardupilot/blob/442e7832146508718b04b5045157b672e4a95780/ArduPlane/mode_qrtl.cpp#L38-L52

Reply:
thanks, fixed